packages/bpk-component-input/src/withOpenEvents.tsx

```diff
@@ -24,11 +24,7 @@ import type {
   ComponentProps,
 } from 'react';
 
-import { cssModules, wrapDisplayName } from '../../bpk-react-utils';
-
-import STYLES from './BpkInput.module.scss';
-
-const getClassName = cssModules(STYLES);
+import { wrapDisplayName } from '../../bpk-react-utils';
```

Comment:
What's this being used for?

Reply:
The `wrapDisplayName` basically will render something like: `withOpenEvents(MyCustomComponent)` when viewed in the react debug tools. 

It's used when defining the `displayName`.Without it we'd loose the knowledge that the component had been wrapped when debugging. 

![Screenshot 2024-05-02 at 10 54 14](https://github.com/Skyscanner/backpack/assets/89925955/5b8c73fc-bca2-4ec1-82f2-5e2c0aecc2ec)

Reply:
Nice I see! That's very cool!